### ISSUE TITLE
Project 'Blazor Webassembly App' option appears twice on create new project screen.

### ISSUE THREAD

**Issue by rxelizondo** [aspnet/AspNetCore#12618]
Not a big deal and I am guessing that you guys are already aware of this, the project 'Blazor Webassembly App' option appears twice on the create new project screen.

![repeat](https://user-images.githubusercontent.com/13452107/61968274-324acd00-af9d-11e9-9f6d-c8dab9bad096.png)

Thanks.

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us, @rxelizondo.
Make sure you uninstall the `Blazor VSIX` extension.
When you install the Blazor project templates via command line - it'll override any other project templates and things will come to proper state.
Anyway, the VSIX is not needed any more - simply uninstall it.

**Comment by rxelizondo** [comment]
@mkArtakMSFT :  My apologies, it looks like when I upgraded to preview 7 I forgot to update to the latest client-side Blazor templates by running:

`dotnet new -i Microsoft.AspNetCore.Blazor.Templates::3.0.0-preview7.19365.7`

After I updated the templates, I no longer see the entry displayed twice, in fact, I no longer see `Blazor Webassembly App `as an option, I only see `Blazor (ASP.NET Core hosted)` listed. Not sure what is exactly is going on but looks like my mistake.

Sorry about that.

Thank you.
